src/parseSerialize/schemas.ts

```diff
@@ -329,6 +329,21 @@ export const setScriptSchemaV1: TSchema = {
   ],
 }
 
+export const transferSchemaV1: TSchema = {
+  type: 'object',
+  schema: [
```

Comment:
Can you put a comment on top, so in the future we know that this schema is not 100% correct, because of a Ledger problem (and ideally attach the issue url posted on Ledger's repo)?

Reply:
Done :)